docker/server/start-workers.sh

```diff
@@ -6,4 +6,5 @@ echo "Waiting for Mongo to start"
 echo "Waiting for Redis to start"
 /usr/local/bin/wait-for-it --strict -t 0 queue:6379
 
-cd /opt/openstudio/server && bundle exec rake environment resque:workers
+# Only start a single worker when calling this script (do not use resque:workers).
```

Comment:
if this is appropriate, we should rename the script (which is start-workers.sh).  but is it used for for worker node setup as well as web-background?

Reply:
Yeah, start-workers.sh only starts a single worker instance and registers it with redis. This was probably causing an issue because in docker-compose.yml (for local use) had COUNT=1 set then called resque:workers. Let me know if you want me to rename before merging.